rpcd/playbooks/verify-maas.yml

```diff
@@ -13,40 +13,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+- include: /opt/rpc-maas/playbooks/maas-verify.yml
+
 - name: Verify MaaS setup
   hosts: hosts
   tasks:
-    - name: ensure remote directory
-      file:
-        path: "{{ maas_rpc_scripts_dir }}"
-        state: directory
-    - name: Copy rpc-maas-tool to all hosts
-      copy:
-        src: "{{ maas_rpc_scripts_dir }}/{{item}}"
-        dest: "{{ maas_rpc_scripts_dir }}/{{item}}"
-        mode: 0755
-      with_items:
-        - rpc-maas-tool.py
-        - alarmparser.py
-
-    - name: "Local MaaS Checks"
-      shell: |
-        . {{ maas_venv_bin }}/activate
-        {{ maas_rpc_scripts_dir }}/rpc-maas-tool.py verify-local
-      register: verify_maas
-      failed_when: verify_maas.rc != 0
-      changed_when: False
-      until: verify_maas.rc == 0
-      retries: "{{ verify_maas_retries }}"
-      delay: "{{ verify_maas_delay }}"
-
     - name: "Verify Checks & Alarms are registered"
-      shell: |
-        . {{ maas_venv_bin }}/activate
-        {{ maas_rpc_scripts_dir }}/rpc-maas-tool.py verify-created \
-        --entity {{ inventory_hostname }}{{ maas_fqdn_extension }} \
-        {% for ec in maas_excluded_checks %} --excludedcheck {{ec}}
-        {% endfor %}
+      shell: >-
```

Comment:
It could be should we choose to refactor rpc-gating. At this time the api tests have to remain here.